packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -38,7 +48,7 @@ export function makeRequestOptions(
 		}
 	}
 
-	const url = /^http(s?):/.test(model) || model.startsWith("/") ? model : `${HF_INFERENCE_API_BASE_URL}${model}`;
+	const url = isModelTypeURL(model) ? model : `${HF_INFERENCE_API_MODEL_BASE_URL}${model}`;
```

Comment:
The `/^http(s?):/.test(model)` test needs to stay, it's for inference endpoints.

Only the latter part of the ternary expression needs to be changed

Reply:
I agree with the previous suggestions, but I'm not getting this one...
I've replaced the first part of the ternary expression with a a function call that just do the same:
```
export function isModelTypeURL(model: string): boolean {
	return /^http(s?):/.test(model) || model.startsWith("/");
}
```

I've done this way in order to not write the same expression in getPipelineURL(...)

Would you please clarify me?
Thanks in advance!

(I'm already implementing the other suggestions you made)